lira/parsers/rst.py

```diff
@@ -23,13 +24,16 @@ class DirectiveNode(Element):
     - content
     - options
     """
-
     tagname = "directive"
 
 
-def importable(value):
-    # TODO: check if value can be imported
-    return value
+def is_importable(value):
+    module_name, class_name = value.rsplit(".",1)
+    MyClass = getattr(importlib.import_module(module_name), class_name, ValueError)
+    if MyClass is ValueError:
```

Comment:
We need to check for `validator_class is not None and isinstance(validator_class, Validator)`